tsconfig.json

```diff
@@ -2,16 +2,20 @@
 	"compilerOptions": {
 		"target": "ES2017",
 		"module": "commonjs",
-		"lib": ["es2017", "dom"],
+		"lib": ["ES2017"],
 		"allowJs": true,
 		"checkJs": true,
 		"noEmit": true,
 		"strict": true,
+		"allowUnusedLabels": false,
+		"allowUnreachableCode": false,
 		"noFallthroughCasesInSwitch": true,
+		"noImplicitReturns": false,
```

Comment:
`noImplicitReturns` - This option can make code readable by promoting explicit `return`, but it doesn't seem reasonable to enable it for now. Because Stylelint is ofter using the following patterns:

```js
root.walkDecls((decl) => {
  if (isSupported(decl)) return;
  if (isAllowed(decl)) return;

  report({ ... });
  // `noImplicitReturns` warns the absence of `return` here!
});
```

Ref: https://www.typescriptlang.org/tsconfig#noImplicitReturns